sample-woo-plugin.php

```diff
@@ -0,0 +1,6 @@
+<?php // phpcs:ignoreFile
```

Comment:
Accidentally added by a previous `copy` iteration?

Reply:
Yup, removed now in https://github.com/woocommerce/woocommerce/pull/26358/commits/28fd8850fa84b42d38d116e7aed35c9d0d537322